.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+### Describe the change
+
+Explanation of what this PR does
+
+### Where should the reviewer start?
+
+Recommendations for how to test this PR
+
+### PR checklist
```

Comment:
I think this is something that we do with labels? I would add as less information as possible, so it is less probable to omit them.

Reply:
Problems with labels is that the developer proactively has to add the label. For me PR checklist is to remind the developer some things that can be forgotten (like a TODO list).